packages/generator/templates/pages/npmrc

```diff
@@ -0,0 +1,11 @@
+save-exact=true
```

Comment:
The file name should be `.npmrc` AFAIK

Reply:
no it should names like this only, the generator will take care of this